example.py

```diff
@@ -12,6 +13,19 @@
     print('\n\n\n')
 
     pyowletClient = PyOwlet(sys.argv[1], sys.argv[2])
+
+    print('Our client is instantiated and should have populated attributes')
+
+    print('Baby Name:' + pyowletClient.baby_name)
+    print('\n\n\n')
+
+    print('These properties will fetch new information automatically if the information is older than 15 seconds.\n')
+    print('You can also bypass the ratelimiting and refresh the properties manually\n')
+
+    pyowletClient.update_properties()
+
+    print('Or query for raw measurements individually\n')
+
     properties = [
         'OXYGEN_LEVEL',
         'HEART_RATE',
```

Comment:
Whats up with the spacing on these lines? 30, 31, and 38 are indented more than 32-37

Reply:
Hmm, not sure I see the difference here

Reply:
oh weird, I see it when I expand the diff, but it looks ok in my editor. I'll have a look

Reply:
hmm weird. Not sure what's up here-- everywhere else this looks ok, even in the committed version when I click on view file on this